source/features/follow-file-renames.tsx

```diff
@@ -5,45 +5,45 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import * as api from '../github-helpers/api';
 import parseRoute from '../github-helpers/parse-route';
+import {getRepoURL} from '../github-helpers';
 
 interface File {
 	previous_filename: string;
 	filename: string;
 	status: string;
 }
 
-async function findRename(
-	user: string,
-	repo: string,
-	lastCommitOnPage: string
-): Promise<File[]> {
+async function findRename(lastCommitOnPage: string): Promise<File[]> {
```

Comment:
Why is there no cache here?

Reply:
Because it's so specific to the current file and commit that seems too rare to cache.